### DISCUSSION TITLE
Dream3d output file suitable for simulating polycrystalline plasticity in Moose

### DISCUSSION THREAD

**Discussion by gaoweijian1** [idaholab/moose#28037] [Q&A Meshing]
Hello, I created a 3D polycrystalline RVE using EBSD data from one direction in dream3d (not based on statistical information, but a 1:1 replication). I want to know what type of file should be output in dream3d in order to input it into moose for crystal plasticity calculation. I have read a lot of discussions and saw some keywords including surface mesh generation and. msh files. Can you tell me the correct operation?
Kind greetings!
Weijian Gao
<img width="227" alt="微信图片_20240702110114" src="https://github.com/idaholab/moose/assets/174400342/763d9721-b73a-4160-b946-9bcd91150f76">

**Comment by GiudGiud** [comment]
Hello

This is the reader you can use in moose
https://mooseframework.inl.gov/source/meshgenerators/EBSDMeshGenerator.html

Everyone seems to be using `.txt` for the file name extension. I am not sure what the exact file type is but you can look through the examples to check

Best,
Guillaume

**Reply by gaoweijian1** [reply]
> Hello 你好
> 
> This is the reader you can use in moose这是您可以在驼鹿中使用的阅读器 https://mooseframework.inl.gov/source/meshgenerators/EBSDMeshGenerator.html
> 
> Everyone seems to be using `.txt` for the file name extension. I am not sure what the exact file type is but you can look through the examples to check似乎每个人都在使用 `.txt` 文件扩展名。我不确定确切的文件类型是什么，但您可以查看示例进行检查
> 
> Best, 最好 Guillaume 纪尧姆

Thank you for your answer!
